lib/RGBot.js

```diff
@@ -486,6 +486,18 @@ class RGBot extends EventEmitter {
             .map(player => player.username)) || []
     }
 
+    /**
+     * Returns a list of enemy usernames (relative to this bot).
+     * @return {string[]} A list of all team name of the opponents.
+     */
+    getOpponentTeamNames() {
+       const myTeam = this.getMyTeam();
+       const otherTeamName = this.matchInfo().teams.find(t => t.name != myTeam)?.name;
+       return (this.#matchInfo && this.#matchInfo.players
+        .filter(player => player.team === otherTeamName)
+        .map(player => player.username)) || []
+    }
```

Comment:
Hey @Theo-flux! Thanks for putting this in. Note that there are a few changes I might suggest here. It seems like your function returns a list of enemy usernames, but this ticket was referring to getting the enemy team names, not the player usernames (for that functionality, I'd check out the existing `getOpponentUsernames()` function.

```suggestion
       const myTeam = this.getMyTeam();
       return this.matchInfo().teams.map(t => t.name).filter(tn => tn !== myTeam);
```

Reply:
ok @vontell, do you want me to wait while you check getOpponentUsernames() function ?

Reply:
Hey @Theo-flux! If you accept that commit suggestion above, that's all that should be left!